upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+Upgrading from Sulu 1.6 to 2.0
+==============================
+
+We have a very extensive `UPGRADE.md file`_ in our repository, which contains almost every breaking change we made
```

Comment:
We provide a ... in the sulu/sulu repository that contains ...

Reply:
Replaced `that`, because for my liking this word already appears way too often in this guide :see_no_evil: